src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -237,9 +246,20 @@ public void periodic() {
             armMotorMaster.set(0);
             armMotorFollower.set(0);
         }
+        
+       
+        
 
         autoCancelArmCommand();
 
+        if(SmartDashboard.getBoolean("babymode", babyMode) == true){
+            armPIDMaster.setOutputRange(-0.4/12, 0.4/12);
+        }
+        else{
+            armPIDMaster.setOutputRange(MIN_VOLTAGE/12, MAX_VOLTAGE/12);
+        }
+
+
```

Comment:
This also needs to be in the constructor so that we aren't calling setOutputRange() constantly.

Reply:
added back to constructor